src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: the radius is")
```

Comment:
The string builder pattern is correct, but the message format is not consistent with English grammar. It should be something like 'Figure: circle, radius: ', followed by the radius value, and then ', color: ', followed by the color.

Reply:
It is consistent with english grammar